src/types/types.ts

```diff
@@ -723,6 +723,35 @@ export interface NftContractTokenBalance {
   balance: number;
 }
 
+/**
+ * Represents an NFT token to fetch metadata for in a {@link NftNamespace.getNftMetadataBatch}
+ * method.
+ */
+export interface NftMetadataBatchToken {
```

Comment:
Should we add tokenType here?

Reply:
I'm going off of the params from the docs: https://docs.alchemy.com/reference/getnftmetadatabatch